modules/appeals_api/config/schemas/10182_headers.json

```diff
@@ -15,7 +15,8 @@
         "X-VA-Veteran-SSN":             { "$ref": "#/definitions/nodCreateHeadersSsn" },
         "X-VA-Veteran-File-Number":     { "$ref": "#/definitions/nodCreateHeadersFileNumber" },
         "X-VA-Veteran-Birth-Date":      { "$ref": "#/definitions/nodCreateHeadersBirthDate" },
-        "X-Consumer-Username":          { "$ref": "#/definitions/nodCreateHeadersConsumerUsername"}
+        "X-Consumer-Username":          { "$ref": "#/definitions/nodCreateHeadersConsumerUsername"},
+        "X-Consumer-ID":                { "$ref": "#/definitions/nodCreateHeadersConsumerId"}
```

Comment:
Is this definitely supposed to be required?

Reply:
It's not required (take note it is not included in the required collection), but supplied by Kong for the stamp on our pdf along with `Consumer-Username`. :)

Reply:
^ Which now makes me wonder if is _needs_ to be required hmmmmm...

Reply:
Update: not required